src/mngt/ocf_mngt_flush.c

```diff
@@ -491,6 +499,8 @@ static void _ocf_mngt_flush_containers(
 	context->fcs.fcnum = fcnum;
 
 	for (i = 0; i < fcnum; i++) {
+		if (fctbl[i].count == 0)
+			continue;
```

Comment:
isn't this re-checking line 254?

Reply:
It is, but `fctbl[i].count` could have been overwritten in line 302